### PR TITLE
Telescope 0.109.0

### PR DESCRIPTION
**Pull request by Zetazzz**
Hoang's changes on fromJSON and fromPartial has been merged
Found pagination fields also are troubled with undefined valued, and I made the fix
upgraded telescope to .109
regen code and passed build

### CODE REVIEW

File: src/cosmos/app/runtime/v1alpha1/module.ts [hunk 1]
```diff
@@ -111,20 +111,15 @@ export const Module = {
...
-      exportGenesis: Array.isArray(object?.exportGenesis)
-        ? object.exportGenesis.map((e: any) => String(e))
-        : [],
-      overrideStoreKeys: Array.isArray(object?.overrideStoreKeys)
-        ? object.overrideStoreKeys.map((e: any) => StoreKeyConfig.fromJSON(e))
-        : [],
-    };
+    const obj = createBaseModule();
+    if (isSet(object.appName)) obj.appName = String(object.appName);
+    if (Array.isArray(object?.beginBlockers)) object.beginBlockers.map((e: any) => String(e));
```
Comment (webmaster128) [inline src/cosmos/app/runtime/v1alpha1/module.ts:116]: Here we are missing an assignment which we had before. The result of the map is lost at the moment.

```suggestion
    if (Array.isArray(object?.beginBlockers)) obj.beginBlockers = object.beginBlockers.map((e: any) => String(e));
```

File: src/cosmos/app/runtime/v1alpha1/module.ts [hunk 2]
```diff
@@ -206,10 +201,10 @@ export const StoreKeyConfig = {
...
   },
   fromJSON(object: any): StoreKeyConfig {
-    return {
-      moduleName: isSet(object.moduleName) ? String(object.moduleName) : "",
-      kvStoreKey: isSet(object.kvStoreKey) ? String(object.kvStoreKey) : "",
-    };
+    const obj = createBaseStoreKeyConfig();
+    if (isSet(object.moduleName)) obj.moduleName = String(object.moduleName);
+    if (isSet(object.kvStoreKey)) obj.kvStoreKey = String(object.kvStoreKey);
+    return obj;
```
Comment (webmaster128) [inline src/cosmos/app/runtime/v1alpha1/module.ts:207]: Looks good

File: src/cosmos/app/v1alpha1/config.ts [hunk 3]
```diff
@@ -124,7 +123,7 @@ export const Config = {
 function createBaseModuleConfig(): ModuleConfig {
   return {
     name: "",
-    config: undefined,
+    config: Any.fromPartial({}),
```
Comment (webmaster128) [inline src/cosmos/app/v1alpha1/config.ts:126]: 💯 this is the protobuf way

File: src/cosmos/app/runtime/v1alpha1/module.ts (hunk 1 above)
Comment (Zetazzz) [reply src/cosmos/app/runtime/v1alpha1/module.ts]: Good catch! Thank you very much!

**Comment by Zetazzz** [comment]
Fixed! Thanks for review!

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+- Upgrade to Telescope 0.105.0. This changes a lot of fields from optional to
```
Comment (webmaster128) [inline CHANGELOG.md:11]:
```suggestion
- Upgrade to Telescope 0.109.1. This changes a lot of fields from optional to
```

**Review by webmaster128 — APPROVED** [review]
Very nice, thank you

**Pull request merged**